openapi-cli/src/main/java/io/ballerina/openapi/cmd/OpenApiCmd.java

```diff
@@ -306,6 +310,7 @@ private String setLicenseHeader() {
      */
     private void generatesClientFile(CodeGenerator generator, String clientName, Path resourcePath, Filter filter) {
         try {
+            generator.setIncludeTestFiles(includeTestFiles);
```

Comment:
Is this essential to set a test flag `setIncludeTestFiles(includeTestFiles)` since it is already set in line 225 at the main command call?